vissl/hooks/byol_hooks.py

```diff
@@ -83,13 +83,13 @@ def _build_byol_target_network(self, task: tasks.ClassyTask) -> None:
         # Restore an hypothetical checkpoint, else random initialize 
         if task.loss.checkpoint is not None:
             task.loss.load_state_dict(task.loss.checkpoint)
-        # else:
-        #     logging.info("Copying model parameters")
-        #     for param_q, param_k in zip(
-        #         task.base_model.parameters(), task.loss.target_network.parameters()
-        #     ):
-        #         param_k.data.copy_(param_q.data)
-        #         param_k.requires_grad = False
+        else:
+            logging.info("Copying model parameters")
+            for param_q, param_k in zip(
+                task.base_model.parameters(), task.loss.target_network.parameters()
```

Comment:
Will this cause issues because the target network does not have the predictor?

Reply:
I will look at this in more detail.

Reply:
yes, Ig will this might upset the saving process, I will get it removed.

Reply:
It's actually OK -- I've tested.